xbmc/interfaces/python/PythonInvoker.h

```diff
@@ -73,4 +84,19 @@ class CPythonInvoker : public ILanguageInvoker
   bool m_systemExitThrown = false;
 
   static CCriticalSection s_critical;
+
+private:
+  struct PyObjectDeleter
+  {
+    void operator()(PyObject* p) const;
+  };
+
+public:
+  typedef std::unique_ptr<PyObject, PyObjectDeleter> PyObjectPtr;
+};
+
+struct PythonModule
+{
+  const char* name;
```

Comment:
can we use `std::string_view` instead?

Reply:
PyImport_AppendInittab() and similar functions don't copy the module name. Using a non-dynamically allocated object like "const chat *" is safer.